docs/RELEASE_WORKFLOW.md

```diff
@@ -1,36 +1,74 @@
-# Release Workflow
+# How to do releases
+
+> We distinct between "test" releases and "official" releases. Test releases
+> serve as "Release Candidates" which do not land in the official app stores
+> yet.
+
+## Test releases
+
+* Before every official release we make at least one test release, similar to
+  "Release Candidates"
+* Test releases are published to supporters and the userbase through our
+  community channels, but not through official app stores
+* Create a forum post to inform about the test release, mention what this
+  release is about and what should be tested, here is an
+  [example](https://support.delta.chat/t/help-testing-the-upcoming-1-41-x-release/2793)
+* The goal is to do more broad user testing, looking for issues which came up
+  and gathering feedback
+* Usually after a round of feedback and bug reports we do another test release
+  and repeat, until the release stabilized
+* Optional: We can use flatpack for test releases as well (they have a
+  "testing" branch users can opt-in)
+* Optional: Draft an in-app device message for the new version informing users
+  of the release highlights and also thanking the testers for testing the app
+
+## Official releases
+
+* Make sure the latest version number is reflected on the official DeltaChat
+  website, adjust the constants in this file:
+  https://github.com/deltachat/deltachat-pages/blob/master/_includes/download-boxes.html
+* An in-app device message for the new official release should exist, if there
+  is no highlight to mention we can say it's a release focused on stability and
+  bug fixes
+* Official releases require individual building steps for each platform we
+  support. You can see an
```

Comment:
It's a checklist of all targets/stores we support, we should make the release progress issue mandatory so everyone can check the progress without asking.

Reply:
To me this document helps a newcomer to understand what they can do to release something. A checklist wouldn't be very sufficient to do that ("what exactly do I have to _do_ for each of these points"). I agree, it's a nice list, but without any actual steps on how to complete it it's useless for a tutorial?

Reply:
Or in other words: If we can specify what to do for each step, I think it makes sense to add it to this document, otherwise I'll just refer to it without much comment